sentry-rails/spec/sentry/rails_spec.rb

```diff
@@ -103,7 +103,7 @@
         expect(response.status).to eq(500)
 
         expect(event["exception"]["values"][0]["type"]).to eq("RuntimeError")
-        expect(event["exception"]["values"][0]["value"]).to eq("An unhandled exception!")
+        expect(event["exception"]["values"][0]["value"]).to match("An unhandled exception!")
```

Comment:
`#detailed_message` always includes `(ExceptionClass)`, which I think is beneficial and should be kept. 
But I also don't want to check Ruby version before every message check, so I just switch to `match` instead.